pandas/tests/frame/test_indexing.py

```diff
@@ -2217,6 +2217,22 @@ def test_reindex_frame_add_nat(self):
         assert mask[-5:].all()
         assert not mask[:-5].any()
 
+    def test_reindex_limit(self):
+        # GH 28631
+        data = [["A", "A", "A"], ["B", "B", "B"], ["C", "C", "C"], ["D", "D", "D"]]
+        exp_data = [
+            ["A", "A", "A"],
+            ["B", "B", "B"],
+            ["C", "C", "C"],
+            ["D", "D", "D"],
+            ["D", "D", "D"],
+            [np.nan, np.nan, np.nan],
+        ]
+        df = pd.DataFrame(data)
```

Comment:
small nitpick but `DataFrame` is already imported so no need for the `pd` prefix here or below.

Reply:
sure!